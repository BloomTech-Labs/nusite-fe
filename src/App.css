.boxedit {
   display: flex;
   flex-flow: column wrap;
   background: whitesmoke;
   border-radius: 2px;
   color: #111;
   justify-content: left;
   align-items: left;
   text-align: left;
   padding: 11px;
}

.dashboard {
   display: flex;
   justify-content: center;
   align-items: center;
   background: whitesmoke;
   border-radius: 2px;
   color: #111;
   text-align: left;
   width: 100%;
}
/* Google */
.loginBtn {
   box-sizing: border-box;
   position: relative;
   /* width: 13em;  - apply for fixed size */
   margin: 0.2em;
   padding: 0 15px 0 46px;
   border: none;
   text-align: left;
   line-height: 34px;
   white-space: nowrap;
   border-radius: 0.2em;
   font-size: 16px;
   color: #fff;
}
.loginBtn:before {
   content: "";
   box-sizing: border-box;
   position: absolute;
   top: 0;
   left: 0;
   width: 34px;
   height: 100%;
}
.loginBtn:hover {
   cursor: pointer;
}
.loginBtn:focus {
   outline: none;
}
.loginBtn:active {
   box-shadow: inset 0 0 0 32px rgba(0, 0, 0, 0.1);
}

.loginBtn--google {
   font-family: "Roboto", arial, sans-serif;
   background: #dd4b39;
}
.loginBtn--google:before {
   border-right: #bb3f30 1px solid;
   background: url("https://s3-us-west-2.amazonaws.com/s.cdpn.io/14082/icon_google.png")
      7px 7px no-repeat;
}
.loginBtn--google:hover,
.loginBtn--google:focus {
   background: #e74b37;
}

.App_dark .edit {
   background: #aaa;
   color: #111;
   border-radius: 2px;
}
