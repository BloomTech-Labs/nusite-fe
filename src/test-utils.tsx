import React from "react";
import { render, RenderResult } from "@testing-library/react";

import { ThemeContext } from "./context/contexts";

interface SpiesMap {
   setDarkMode: jest.Mock;
}

interface RenderWithSpiesResult extends RenderResult {
   mockSpies: SpiesMap;
}

const mockSpies: SpiesMap = {
   setDarkMode: jest.fn(),
};

const AllTheProviders: React.FC = ({ children }) => (
   <ThemeContext.Provider
      value={{ darkMode: false, setDarkMode: mockSpies.setDarkMode }}
   >
      {children}
   </ThemeContext.Provider>
);

const customRender = (ui: React.ReactElement, options?: Object | undefined) => {
   return {
      ...render(ui, { wrapper: AllTheProviders, ...options }),
      mockSpies,
   };
};

export * from "@testing-library/react";
export { customRender as render, RenderWithSpiesResult };
