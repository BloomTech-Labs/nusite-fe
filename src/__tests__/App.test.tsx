import React from "react";
import ReactDOM from "react-dom";
import { render, cleanup, RenderWithSpiesResult } from "../test-utils";
import "@testing-library/jest-dom/extend-expect";

import App from "../App";
afterEach(cleanup);

test("App renders without crashing", (): void => {
   const div: HTMLDivElement = document.createElement("div");
   ReactDOM.render(<App />, div);
   ReactDOM.unmountComponentAtNode(div);
});

test("Home page renders correctly", (): void => {
   const app: RenderWithSpiesResult = render(<App />);
   expect(app).toMatchSnapshot();
});
