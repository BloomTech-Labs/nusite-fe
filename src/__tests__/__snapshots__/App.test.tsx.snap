// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="App"
        >
          <header
            class="App"
          >
            <h1>
              PartNerd 
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 18c1.1 0 1.99-.9 1.99-2L22 5c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2H0c0 1.1.9 2 2 2h20c1.1 0 2-.9 2-2h-4zM4 5h16v11H4V5zm8 14c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1z"
                />
              </svg>
            </h1>
            <nav
              color="white"
            >
              <a
                aria-current="page"
                class="active"
                href="#/"
              >
                Home
              </a>
              <a
                href="#/home"
              >
                Dashboard
              </a>
              <a
                href="#/login"
              >
                Login
              </a>
              <a
                href="#/register"
              >
                Register
              </a>
            </nav>
            <div>
              <svg
                aria-controls="simple-menu"
                aria-haspopup="true"
                aria-hidden="true"
                class="MuiSvgIcon-root menu"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                />
              </svg>
            </div>
          </header>
          <main>
            <br />
            <span
              class="darkMode"
            >
              Click for Dark Mode
            </span>
            <br />
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-141 MuiCheckbox-root MuiCheckbox-colorSecondary darkMode MuiIconButton-colorSecondary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-144"
                  data-indeterminate="false"
                  type="checkbox"
                  value="false"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
            <br />
            <br />
            <h1
              class="tagline"
            >
              Connecting web developers to people with great ideas. Old website, new website, app, or feature, we've got you covered.
            </h1>
            <br />
            <p>
              We are creating a marketplace for developers and project owners/clients to meet each other and form partnerships. Developers and project owners can offer their services and projects to one another and be matched according to their needs.
            </p>
            <br />
            <br />
          </main>
          <div
            class="parallax"
          >
            <br />
            <br />
            <h2
              class="tagline-2"
            >
              Comprehensive recommendation based on advanced data analysis.
            </h2>
          </div>
          <br />
          <section>
            <br />
            <h2>
              Project Builders
            </h2>
            <br />
            <img
              alt=""
              class="avatars"
              src="client.jpg"
            />
            <br />
            <p>
              A marketplace for developers and project owners to meet each other and form partnerships. Developers and project owners can offer their services and projects to one another and be matched according to their needs.
            </p>
            <br />
            <br />
            <br />
            <div
              class="gif-wrapper"
            >
              <img
                alt=""
                class="user-flow"
                src="userflow.gif"
              />
            </div>
            <br />
            <br />
            <br />
            <p>
              Find someone to build, fix, or add on to your existing site or idea in minutes. Just set up a profile and browse existing projects to get and idea of what you like, pick someone, and/or just let us give your best matches for your project based on developer skills. Take the workload off of yourself and get started today. It's easy.
            </p>
            <br />
            <br />
            <br />
            <hr />
            <br />
            <h4
              class="tagline"
            >
              PartNerdâ„¢ provides users a comprehensive recommendation generation based on advanced data analysis.
              <br />
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained start-main MuiButton-containedSecondary"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  <a
                    href="#/register"
                  >
                    Get Started
                  </a>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </h4>
            <br />
            <div
              class="google-btn"
            >
              <div
                class="google-icon-wrapper"
              >
                <img
                  alt=""
                  class="google-icon"
                  src="https://upload.wikimedia.org/wikipedia/commons/5/53/Google_%22G%22_Logo.svg"
                />
              </div>
              <a
                href="https://google.com/"
              >
                <p
                  class="btn-text"
                >
                  <b>
                    Google Signup 
                  </b>
                </p>
              </a>
            </div>
            <br />
            <div
              class="github-btn"
            >
              <div
                class="github-icon-wrapper"
              >
                <img
                  alt=""
                  class="github-icon"
                  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAAh1BMVEX///8YFhYAAAAQDQ0UEhJEQ0MKBgYTERHz8/P6+vpiYWFUU1O0tLQeHBwRDg4qKSmjoqLi4uLT09NJSEhpaGglIyPHxsaYl5eAgIA+PT14d3eJiIggHh7Nzc3d3d3U1NQxMDDr6+uura2Uk5NubW0wLi6EhIS7u7tjYmI4NzdOTU2WlpZaWVlxXGWoAAAIFElEQVR4nO2daZubvA6GB5lxICvZE5JZskynaef//75DdsAWEGLZ9D26P/VqU8wTW5Ys2c7LC8MwDMMwDMMwDMMwDMMwDMMwDMMwDMMwDMMwDMP8HxO6fgEaglF3vmiPD4PpureeDg7j9mLe/Qhcv5YZgtm8DSek6Lf8I62+kOe/+tnN/nGZb6vlUZrv6fGPQperN9evWZdZnLy/QMTdEcmn9jPXL/s474ukf7C+U/oyEbn4cP3KD9EdV5d3E/nadf3aVQkjUWFwqiT/K/onfEmi77HuS3ekjFy/fildWVvfRWOzx+po+ZS+s8blyLUMlDCuZX95BMQNNceZBAP6jgA00j8uoGVI4HGoxq7lKAwHpjrwDEyHriVl6RqxwDQCGjWprp6eQlV8WLmWdadtdoRegbZrYRfCVxqBicRxI9xG8EUlMJH42YAFcjCVZAI9T06dS6QVmEgcOJYYftEKTCR+urVFsknmDoxdCvxNLzCROHEncGVDYCJx50rgtx2BicRvNwKHSqjmn9K8zywxzk9Qn+smDB/kg21/OnubbRdfVbKkOvoAvThKnjHNSxSfLgQulDF6tZfNdgyPD2AJX/NLV6n27SIKn6ka4J7Ufd8/qFFC557c/9A823ryJpSKufnr9Ac2MWiCgVNlRv1rAe339H/uKZ9p9ezq041RT+b81rB9m3R8AXAtQMnrH8VVRgvGuZT+b/XLsT1ONePIAyWbO5vCub7U66yi2Wh4DTGD4WgWrTq9c10KPMUZRLrHv+c/RcqrZrYETZVsAetJNMKC52AUTdawUP/hTaNQdowqKKGrm0a0TmtT+izdJ4ba59tMMWrTMmBuERDqFLYGxp5fis5MEoUGW9A3YC37FgptZo1cob1O1HchvUJ7EXhLnxw1qFBrh0lgsDTXRBHYognK582qaOdST++QCND5QsPN6/zhkXzYRMM7FlKrMU1t5mgbNjJvv7DWDQYdHWyBafBbxMGLMMYMcYOuvMSroSYK0KwLbwpNhVVFbdDnM2K0dfhlrBHUDm0MU3SQSpMDaIJJpHeJ2Dzu+QbdYeLy11jGjnw23aHz+NxoO6gpkkduS2QeN55H6SAlH92C2SQB+tVuDbeky5Mc6RMbIjZ4fGG8qTEWHNIa4l8sJDafCdNmSjzy6LuNWYf5RFiA+CVij4i0SrL6RoJT2g1h2ERjMJy5gwQ2tFMNNsGROCksuDCZLVFAl/cU8TC2wjCYtVTBImKarxVrjDK9r6nHHPFpCkNTfWxKWmhDnAXRuhTx+aSRKdKm/CFp7Uf/fRoPENMc+nqFv0laQ0YMqcsf6C3Dch+aXadlUTZJnBE0pT0sqPlL0tqJUK2vnyAKM5Z6myBVqNtl4OX3KBhjjYTehKMU60PLHp/SDpFvlSaQQgpQTuZSmkAKLUA58Ic0mwiwjAlpTIMW1ihMAyk10+7JwJIYJEHN3l7G5A5WWSNxF6jRm0yu58Erl+aXwGiJzckan2IG3yJttWjCiwtYnsYTf4y3hcTdniQ974Um9c0bB14/oN2Gqd8N5RGcGMB8BXXxaYIdAvJN2z9eQKTdZ1pQfjbr9LGqhecL2nNQaAnY86XRmhDmDMk30uJTjdl6Albjol3/nviDHxYxGC9iywrPwrEE3BCTycbYOP1C1jAEM5oC6vMT5MFQIwVnw4kSl2n6BSfvwcwsgG9KsrITGildXNo3cby8SKCNzYmjwvNMMHg2egs7RQ3QBqUXkIrQBfFkFmXUK/4Gbez0RndFnfGhU3+tGJRcAeNLGwe7s76qJUHmpnYJcT1jCf+WHemj3hB1Ib1wg+lktT/kz7UC7B+/X264Kr/jzdJp2VSW77KzfDPPf/kSPnePrAGGUZU77Kwd7vq8Dctb0Slc5N+vDyDibRWVw+5iXe3osLWzXalszT3c/la74HQwexxHXfR0XjeKx9WPf9s6UPKS7kQPllfT+NCOsuPpUXz5P0mdJK3QhfaO5adTbuKWZkPWjoV5lU80xFax2IUvL4e01dymcO26QxSG48URUu6rsnmTa7a7brvadBuIS94LqxOoWAnY7mT30l8jNc2BoX7Jiqp6J1q+OWKTnVauravjtDQZjubU8g+yfatCNp15O2uhrHxKc38Vb4DpT6kVKSwzBnRbmMb5AK4sVC44AZR5jv0Lo7N3t9wPI2xT0Vfi8kuvQMJPcWUEUmzRLSNbG7pnF4LV9VoIaMUVEmNVFAqLB9VTZPJF/a/Uv3xEq1+r3Xe1yQ/N/d5xdQNPmLmCp7ahIDuf9APEMhlTFHUvHcN2P6QEujDCM5l5sG4ZuFQh0Ox8rEbGK9Yse5UplAenl+6l/XXN+aBEofAdX5y4T0ls1ZptihWKPuXekkrsM71Yo9ZdqFA48hMZ9hlbXD88sRcpFF4DBOai7Rb4i9QvqwzLc0cFCmXP+RA9k82Cn/JPr5M4nvzpAZQX3HCFsGzENcJHtvkklH+8s0wKv8pZE1Qh0JcKqzPS3T53MqTaCht1nXfCBrmMtrZC2bwfD1CS3k8phNeGzDFpZn1NN9ZTKEi349cnmKjZ+ToKfVjavUDwAWZKAbeGQrByy05tdrky28MKa5dXrbGZZBzHgwrlMyVya7y3U/34kEKAn3/kp8mOV+xeXEeF3MalaN4C2Dd2glEJdv55sFbYsHjcSZrEsoN5w+1PYXb6XUe/guNeJSHsorm/YlVA8B1VK9qGDYxfGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhmP8q/wOa7F1sPxWUkQAAAABJRU5ErkJggg=="
                />
              </div>
              <a
                href="https://github.com/"
              >
                <p
                  class="btn-text"
                >
                  <b>
                    Github Signup 
                  </b>
                </p>
              </a>
            </div>
            <br />
            <br />
            <hr />
            <br />
            <h2>
              For Developers 
            </h2>
            <br />
            <p>
              Register as a developer, set up a brief profile, show off your skills and portfolio as they continue to grow, and get paid projects now. It's that simple, no hiring managers, pick projects and let the projects you want get matched with you today.
               
            </p>
            <br />
            <br />
          </section>
          <section
            class="center"
          >
            <div
              id="social-test"
            >
              <div
                class="social"
              >
                <div>
                  <li>
                    <i
                      aria-hidden="true"
                      class="fa fa-facebook"
                    />
                  </li>
                  <li>
                    <i
                      aria-hidden="true"
                      class="fa fa-twitter"
                    />
                  </li>
                  <li>
                    <i
                      aria-hidden="true"
                      class="fa fa-github"
                    />
                  </li>
                  <li>
                    <i
                      aria-hidden="true"
                      class="fa fa-linkedin-square"
                    />
                  </li>
                </div>
              </div>
            </div>
          </section>
          <footer>
            <nav>
              <a
                href="https://partnerd.dev"
              >
                Home
              </a>
              <a
                href="https://partnerd.dev"
              >
                Contact
              </a>
              <a
                href="https://partnerd.dev"
              >
                Login
              </a>
              <a
                href="https://partnerd.dev"
              >
                Register
              </a>
            </nav>
            <br />
            <h4>
              A web development marketplace
            </h4>
            <br />
            <h4>
              Â© PartNerd 2020
            </h4>
            <br />
          </footer>
        </div>
      </div>
    </div>
    <div
      aria-hidden="true"
      class="MuiPopover-root"
      id="simple-menu"
      role="presentation"
      style="position: fixed; z-index: 1300; right: 0px; bottom: 0px; top: 0px; left: 0px; visibility: hidden;"
    >
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiPaper-root MuiMenu-paper MuiPopover-paper MuiPaper-elevation8 MuiPaper-rounded"
        style="opacity: 0; transform: scale(0.75, 0.5625); visibility: hidden;"
        tabindex="-1"
      >
        <ul
          class="MuiList-root MuiMenu-list MuiList-padding"
          role="menu"
          tabindex="-1"
        >
          <li
            aria-disabled="false"
            class="MuiButtonBase-root MuiListItem-root MuiMenuItem-root MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
            role="menuitem"
            tabindex="0"
          >
            Profile
            <span
              class="MuiTouchRipple-root"
            />
          </li>
          <li
            aria-disabled="false"
            class="MuiButtonBase-root MuiListItem-root MuiMenuItem-root MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
            role="menuitem"
            tabindex="-1"
          >
            My account
            <span
              class="MuiTouchRipple-root"
            />
          </li>
          <li
            aria-disabled="false"
            class="MuiButtonBase-root MuiListItem-root MuiMenuItem-root MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
            role="menuitem"
            tabindex="-1"
          >
            Logout
            <span
              class="MuiTouchRipple-root"
            />
          </li>
        </ul>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="App"
      >
        <header
          class="App"
        >
          <h1>
            PartNerd 
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 18c1.1 0 1.99-.9 1.99-2L22 5c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2H0c0 1.1.9 2 2 2h20c1.1 0 2-.9 2-2h-4zM4 5h16v11H4V5zm8 14c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1z"
              />
            </svg>
          </h1>
          <nav
            color="white"
          >
            <a
              aria-current="page"
              class="active"
              href="#/"
            >
              Home
            </a>
            <a
              href="#/home"
            >
              Dashboard
            </a>
            <a
              href="#/login"
            >
              Login
            </a>
            <a
              href="#/register"
            >
              Register
            </a>
          </nav>
          <div>
            <svg
              aria-controls="simple-menu"
              aria-haspopup="true"
              aria-hidden="true"
              class="MuiSvgIcon-root menu"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </div>
        </header>
        <main>
          <br />
          <span
            class="darkMode"
          >
            Click for Dark Mode
          </span>
          <br />
          <span
            aria-disabled="false"
            class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-141 MuiCheckbox-root MuiCheckbox-colorSecondary darkMode MuiIconButton-colorSecondary"
          >
            <span
              class="MuiIconButton-label"
            >
              <input
                class="PrivateSwitchBase-input-144"
                data-indeterminate="false"
                type="checkbox"
                value="false"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </span>
          <br />
          <br />
          <h1
            class="tagline"
          >
            Connecting web developers to people with great ideas. Old website, new website, app, or feature, we've got you covered.
          </h1>
          <br />
          <p>
            We are creating a marketplace for developers and project owners/clients to meet each other and form partnerships. Developers and project owners can offer their services and projects to one another and be matched according to their needs.
          </p>
          <br />
          <br />
        </main>
        <div
          class="parallax"
        >
          <br />
          <br />
          <h2
            class="tagline-2"
          >
            Comprehensive recommendation based on advanced data analysis.
          </h2>
        </div>
        <br />
        <section>
          <br />
          <h2>
            Project Builders
          </h2>
          <br />
          <img
            alt=""
            class="avatars"
            src="client.jpg"
          />
          <br />
          <p>
            A marketplace for developers and project owners to meet each other and form partnerships. Developers and project owners can offer their services and projects to one another and be matched according to their needs.
          </p>
          <br />
          <br />
          <br />
          <div
            class="gif-wrapper"
          >
            <img
              alt=""
              class="user-flow"
              src="userflow.gif"
            />
          </div>
          <br />
          <br />
          <br />
          <p>
            Find someone to build, fix, or add on to your existing site or idea in minutes. Just set up a profile and browse existing projects to get and idea of what you like, pick someone, and/or just let us give your best matches for your project based on developer skills. Take the workload off of yourself and get started today. It's easy.
          </p>
          <br />
          <br />
          <br />
          <hr />
          <br />
          <h4
            class="tagline"
          >
            PartNerdâ„¢ provides users a comprehensive recommendation generation based on advanced data analysis.
            <br />
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained start-main MuiButton-containedSecondary"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                <a
                  href="#/register"
                >
                  Get Started
                </a>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </h4>
          <br />
          <div
            class="google-btn"
          >
            <div
              class="google-icon-wrapper"
            >
              <img
                alt=""
                class="google-icon"
                src="https://upload.wikimedia.org/wikipedia/commons/5/53/Google_%22G%22_Logo.svg"
              />
            </div>
            <a
              href="https://google.com/"
            >
              <p
                class="btn-text"
              >
                <b>
                  Google Signup 
                </b>
              </p>
            </a>
          </div>
          <br />
          <div
            class="github-btn"
          >
            <div
              class="github-icon-wrapper"
            >
              <img
                alt=""
                class="github-icon"
                src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAAh1BMVEX///8YFhYAAAAQDQ0UEhJEQ0MKBgYTERHz8/P6+vpiYWFUU1O0tLQeHBwRDg4qKSmjoqLi4uLT09NJSEhpaGglIyPHxsaYl5eAgIA+PT14d3eJiIggHh7Nzc3d3d3U1NQxMDDr6+uura2Uk5NubW0wLi6EhIS7u7tjYmI4NzdOTU2WlpZaWVlxXGWoAAAIFElEQVR4nO2daZubvA6GB5lxICvZE5JZskynaef//75DdsAWEGLZ9D26P/VqU8wTW5Ys2c7LC8MwDMMwDMMwDMMwDMMwDMMwDMMwDMMwDMMwDMP8HxO6fgEaglF3vmiPD4PpureeDg7j9mLe/Qhcv5YZgtm8DSek6Lf8I62+kOe/+tnN/nGZb6vlUZrv6fGPQperN9evWZdZnLy/QMTdEcmn9jPXL/s474ukf7C+U/oyEbn4cP3KD9EdV5d3E/nadf3aVQkjUWFwqiT/K/onfEmi77HuS3ekjFy/fildWVvfRWOzx+po+ZS+s8blyLUMlDCuZX95BMQNNceZBAP6jgA00j8uoGVI4HGoxq7lKAwHpjrwDEyHriVl6RqxwDQCGjWprp6eQlV8WLmWdadtdoRegbZrYRfCVxqBicRxI9xG8EUlMJH42YAFcjCVZAI9T06dS6QVmEgcOJYYftEKTCR+urVFsknmDoxdCvxNLzCROHEncGVDYCJx50rgtx2BicRvNwKHSqjmn9K8zywxzk9Qn+smDB/kg21/OnubbRdfVbKkOvoAvThKnjHNSxSfLgQulDF6tZfNdgyPD2AJX/NLV6n27SIKn6ka4J7Ufd8/qFFC557c/9A823ryJpSKufnr9Ac2MWiCgVNlRv1rAe339H/uKZ9p9ezq041RT+b81rB9m3R8AXAtQMnrH8VVRgvGuZT+b/XLsT1ONePIAyWbO5vCub7U66yi2Wh4DTGD4WgWrTq9c10KPMUZRLrHv+c/RcqrZrYETZVsAetJNMKC52AUTdawUP/hTaNQdowqKKGrm0a0TmtT+izdJ4ba59tMMWrTMmBuERDqFLYGxp5fis5MEoUGW9A3YC37FgptZo1cob1O1HchvUJ7EXhLnxw1qFBrh0lgsDTXRBHYognK582qaOdST++QCND5QsPN6/zhkXzYRMM7FlKrMU1t5mgbNjJvv7DWDQYdHWyBafBbxMGLMMYMcYOuvMSroSYK0KwLbwpNhVVFbdDnM2K0dfhlrBHUDm0MU3SQSpMDaIJJpHeJ2Dzu+QbdYeLy11jGjnw23aHz+NxoO6gpkkduS2QeN55H6SAlH92C2SQB+tVuDbeky5Mc6RMbIjZ4fGG8qTEWHNIa4l8sJDafCdNmSjzy6LuNWYf5RFiA+CVij4i0SrL6RoJT2g1h2ERjMJy5gwQ2tFMNNsGROCksuDCZLVFAl/cU8TC2wjCYtVTBImKarxVrjDK9r6nHHPFpCkNTfWxKWmhDnAXRuhTx+aSRKdKm/CFp7Uf/fRoPENMc+nqFv0laQ0YMqcsf6C3Dch+aXadlUTZJnBE0pT0sqPlL0tqJUK2vnyAKM5Z6myBVqNtl4OX3KBhjjYTehKMU60PLHp/SDpFvlSaQQgpQTuZSmkAKLUA58Ic0mwiwjAlpTIMW1ihMAyk10+7JwJIYJEHN3l7G5A5WWSNxF6jRm0yu58Erl+aXwGiJzckan2IG3yJttWjCiwtYnsYTf4y3hcTdniQ974Um9c0bB14/oN2Gqd8N5RGcGMB8BXXxaYIdAvJN2z9eQKTdZ1pQfjbr9LGqhecL2nNQaAnY86XRmhDmDMk30uJTjdl6Albjol3/nviDHxYxGC9iywrPwrEE3BCTycbYOP1C1jAEM5oC6vMT5MFQIwVnw4kSl2n6BSfvwcwsgG9KsrITGildXNo3cby8SKCNzYmjwvNMMHg2egs7RQ3QBqUXkIrQBfFkFmXUK/4Gbez0RndFnfGhU3+tGJRcAeNLGwe7s76qJUHmpnYJcT1jCf+WHemj3hB1Ib1wg+lktT/kz7UC7B+/X264Kr/jzdJp2VSW77KzfDPPf/kSPnePrAGGUZU77Kwd7vq8Dctb0Slc5N+vDyDibRWVw+5iXe3osLWzXalszT3c/la74HQwexxHXfR0XjeKx9WPf9s6UPKS7kQPllfT+NCOsuPpUXz5P0mdJK3QhfaO5adTbuKWZkPWjoV5lU80xFax2IUvL4e01dymcO26QxSG48URUu6rsnmTa7a7brvadBuIS94LqxOoWAnY7mT30l8jNc2BoX7Jiqp6J1q+OWKTnVauravjtDQZjubU8g+yfatCNp15O2uhrHxKc38Vb4DpT6kVKSwzBnRbmMb5AK4sVC44AZR5jv0Lo7N3t9wPI2xT0Vfi8kuvQMJPcWUEUmzRLSNbG7pnF4LV9VoIaMUVEmNVFAqLB9VTZPJF/a/Uv3xEq1+r3Xe1yQ/N/d5xdQNPmLmCp7ahIDuf9APEMhlTFHUvHcN2P6QEujDCM5l5sG4ZuFQh0Ox8rEbGK9Yse5UplAenl+6l/XXN+aBEofAdX5y4T0ls1ZptihWKPuXekkrsM71Yo9ZdqFA48hMZ9hlbXD88sRcpFF4DBOai7Rb4i9QvqwzLc0cFCmXP+RA9k82Cn/JPr5M4nvzpAZQX3HCFsGzENcJHtvkklH+8s0wKv8pZE1Qh0JcKqzPS3T53MqTaCht1nXfCBrmMtrZC2bwfD1CS3k8phNeGzDFpZn1NN9ZTKEi349cnmKjZ+ToKfVjavUDwAWZKAbeGQrByy05tdrky28MKa5dXrbGZZBzHgwrlMyVya7y3U/34kEKAn3/kp8mOV+xeXEeF3MalaN4C2Dd2glEJdv55sFbYsHjcSZrEsoN5w+1PYXb6XUe/guNeJSHsorm/YlVA8B1VK9qGDYxfGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhmP8q/wOa7F1sPxWUkQAAAABJRU5ErkJggg=="
              />
            </div>
            <a
              href="https://github.com/"
            >
              <p
                class="btn-text"
              >
                <b>
                  Github Signup 
                </b>
              </p>
            </a>
          </div>
          <br />
          <br />
          <hr />
          <br />
          <h2>
            For Developers 
          </h2>
          <br />
          <p>
            Register as a developer, set up a brief profile, show off your skills and portfolio as they continue to grow, and get paid projects now. It's that simple, no hiring managers, pick projects and let the projects you want get matched with you today.
             
          </p>
          <br />
          <br />
        </section>
        <section
          class="center"
        >
          <div
            id="social-test"
          >
            <div
              class="social"
            >
              <div>
                <li>
                  <i
                    aria-hidden="true"
                    class="fa fa-facebook"
                  />
                </li>
                <li>
                  <i
                    aria-hidden="true"
                    class="fa fa-twitter"
                  />
                </li>
                <li>
                  <i
                    aria-hidden="true"
                    class="fa fa-github"
                  />
                </li>
                <li>
                  <i
                    aria-hidden="true"
                    class="fa fa-linkedin-square"
                  />
                </li>
              </div>
            </div>
          </div>
        </section>
        <footer>
          <nav>
            <a
              href="https://partnerd.dev"
            >
              Home
            </a>
            <a
              href="https://partnerd.dev"
            >
              Contact
            </a>
            <a
              href="https://partnerd.dev"
            >
              Login
            </a>
            <a
              href="https://partnerd.dev"
            >
              Register
            </a>
          </nav>
          <br />
          <h4>
            A web development marketplace
          </h4>
          <br />
          <h4>
            Â© PartNerd 2020
          </h4>
          <br />
        </footer>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
